### ISSUE TITLE
Torch and Tensorflow lambda stuck at 1.0

### ISSUE THREAD

**Issue by rickstaa** [rickstaa/LAC-TF2-TORCH-translation#38]
When using the new algorithms (PyTorch and TF2) the lambda sometimes gets stuck at 1.0 and stays there for the whole training. This does not happen in the original TF1 script.

# How to reproduce

1. Run the train.py script (for multiple policies) in the LAC_TORCH or LAC_TF2 folder.
2. In some of the policies the lambda will not decrease. The behaviour will be as follows:

![image](https://user-images.githubusercontent.com/17570430/101015264-6ce85080-3567-11eb-99fd-590e055f7dd6.png)

A correct run shows the following result:

![image](https://user-images.githubusercontent.com/17570430/101015616-f6981e00-3567-11eb-8e21-67840c0f3d58.png)

# Hypothesis 1

When we compare a run in which lambda is decreasing with a run in which lambda is higher than one (lambda is capped at 1.0) we see some interesting things:
- Actor loss is higher
- Entropy is higher
- Lyapunov error is higher

It looks therefore to me that the algorithm in these cases just ends up in a suboptimal local optimum and it needs more time to get out of this. If this is the case the lambda would decrease if I increase the number of steps to `1e6`. Further, this behaviour should also be observed in the original algorithm when I check out more random seeds. Another possible cause could be that we only cap the alpha between 1.0 while the log alpha is not caped. It might be that in Minghoas code both are capped due to the TF1 works.

# Testing 1

The hypothesis above is not correct on the following results:

- I haven't encountered the problem with the old LAC code even after training 50 policies.
    - The Lyapunov error is not always higher so this is probably not the cause.
- The lambda stays on 1.0 even after `1e6` steps.
